pandas/core/interchange/column.py

```diff
@@ -314,7 +322,12 @@ def _get_data_buffer(
             # Define the dtype for the returned buffer
             # TODO: this will need correcting
             # https://github.com/pandas-dev/pandas/issues/54781
-            dtype = self.dtype
+            dtype = (
+                DtypeKind.UINT,
+                8,
+                ArrowCTypes.UINT8,
+                Endianness.NATIVE,
+            )  # note: currently only support native endianness
```

Comment:
Are we planning to ever support anything but that as an exporter?

Reply:
I don't know tbh

cc @jorisvandenbossche in case you have comments on this